### PR TITLE
Choose a better name for a template argument.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

**Comment by bangerth** [comment]
Choices for `T` are, by the way, `Description` and `DescriptionTemp`. I don't yet know what they refer to, but `T` seems an overly generic name for the template argument :-)

File: source/grid/tria_description.cc [hunk 1]
```diff
@@ -442,11 +442,11 @@ namespace TriangulationDescription
...
         }
 
-        template <typename T>
-        T
+        template <typename DescriptionType>
+        DescriptionType
         create_description_for_rank(const unsigned int my_rank) const
         {
-          T construction_data;
+          DescriptionType construction_data;
```
Comment (drwells) [inline source/grid/tria_description.cc:449]: Could you add something like

```cpp
static_assert(std::is_same_v<DescriptionType, Description<dim, spacedim> || std::is_same_v<DescriptionType, DescriptionTemp<dim, spacedim>);
```
here to better document which types this works with?

Comment (bangerth) [reply source/grid/tria_description.cc:449]: I put that in a follow-up patch that I'll submit if you were willing to merge this as is.

Comment (drwells) [reply source/grid/tria_description.cc:449]: Sounds good!

**Review by drwells — APPROVED** [review]

**Pull request merged**